apps/cricket-score-frontend/README.md

```
@@ -111,10 +113,10 @@
 
 ### Installation
 
-1. **Clone the repository**
+1. **Clone the monorepo**
    ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)